senator/__main__.py

```diff
@@ -1,15 +1,49 @@
 from senator.utility import *
 from senator.game import *
+import argparse
+import numpy as np
 
-OWNER_LOSS = 0.59
-OTHER_LOSS = 0.19
-OWNER_TRASHOLD = 0.18 #Set 2.0 to change the model TODO: find better solution than 2.0
 
+def main() -> tuple[np.ndarray, np.ndarray, np.ndarray]:
 
-votes, initial_utility, is_owner = parse_data(load_data(FILE_PATH))
-game = Game(initial_utility, votes, is_owner, OWNER_LOSS, OTHER_LOSS, OWNER_TRASHOLD)
-#game = Game(np.zeros((3,17)),votes)
+    parser = argparse.ArgumentParser(
+        description="Run the senator game simulation with specified parameters."
+    )
 
-actions_in_steps, final_utility, strategies = game.run_greedy_search()
-print(strategies)
-#print(final_utility.sum(axis=1))
+    parser.add_argument(
+        "--owner_loss",
+        type=float,
+        default=0.8,
+        help="Loss for the owner (default: 0.5)",
```

Comment:
Ty help messages jsou k tomu aby když dáš `python -m senator --help` se vyprintily všechny možný optiony a informace o nich. Ty tam teď dáváš i defaultní hodnoty. To je ok, ale třeba tady jsi očividně ten default změnil, a pak neupdatoval ten help.

Aby se tohle nestávalo, můžeš tomu argparseru říct ať to dělá sám. To se dělá tak, že mu nastavíš help formatter, jinými slovy tu definici `parser`u změníš na 
```python
    parser = argparse.ArgumentParser(
        description="Run the senator game simulation with specified parameters.",
        formatter_class=argparse.ArgumentDefaultsHelpFormatter
    )
```

Reply:
HOHOHO! Done